app/helpers/sellers_helper.rb

```diff
@@ -4,9 +4,20 @@ module SellersHelper
   def self.generate_seller_json(seller:)
     locations = seller.locations
     distributor = seller.distributor
+    fees = seller.fees
+
     json = seller.as_json
     json['distributor'] = distributor.as_json unless distributor.nil?
     json['locations'] = locations.as_json
+    json['fees'] = fees.as_json
+
+    # Take into account the associated fees when calculating the cost per meal
+    # for the customer. eg) We might generate a $10 gift card for GAM, but if
+    # there is a 10% fee associated with creating that gift card, then we should
```

Comment:
Question about flow: So I'm guessing that we'll be displaying the fee amount on the front-end to the user--would the charged amount be the with-fee or without-fee amount?

If it's the with-fee amount, then do we need to do this math here, or should we be doing it on the front-end?

Reply:
Good question. I prefer making the frontend dumb, and doing the math on the backend to reduce potential errors. As for whether or not we're displaying the fee amount on the frontend, I'm not sure. I think that to the user, as long as they know that their $11 gets a meal to the end user, that's all that matters to them. The details of the fact that 2.75% goes to cc processing, and 7.25% goes to Think!Chinatown — might just confuse the end user. 

What do you think as a user? Also @nanxiy might have an opinion here too

Reply:
Bumping, @nanxiy do you think we *need* to show the fees charged per meal? or is it cool to keep it simple and just say that $11 buys a meal! (even if that meal is a $10 meal)